File: Api/MySerialList.Data/Model/FilmProduction.cs
using System;
using System.Collections.Generic;

namespace MySerialList.Data.Model
{
    public class FilmProduction
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Released { get; set; }
        public string Genre { get; set; }
        public string Director { get; set; }
        public string Actors { get; set; }
        public string Plot { get; set; }
        public string Language { get; set; }
        public string Poster { get; set; }
        public bool IsSeries { get; set; }

        public virtual IEnumerable<Episode> Episodes { get; set; }
        public virtual IEnumerable<ReviewFilmProduction> Reviews { get; set; }
        public virtual IEnumerable<Comment> Comments { get; set; }
    }
}

File: my_serial_list/lib/features/film_production/domain/entities/film_production.dart
import 'package:meta/meta.dart';

class FilmProduction {
  final int filmProductionId;
  final String title;
  final String released;
  final String genre;
  final String director;
  final String actors;
  final String plot;
  final String language;
  final String poster;
  final int votes;
  final double rating;
  final bool isSeries;
  final int episodes;
  final int myRating;
  final bool isInMyList;

  FilmProduction({
    @required this.filmProductionId,
    @required this.title,
    @required this.released,
    @required this.genre,
    @required this.director,
    @required this.actors,
    @required this.plot,
    @required this.language,
    @required this.poster,
    @required this.votes,
    @required this.rating,
    @required this.isSeries,
    @required this.episodes,
    @required this.myRating,
    @required this.isInMyList,
  });
}
